addons/test_xlsx_export/tests/test_export.py

```diff
@@ -116,13 +116,13 @@ def test_float_min(self):
 
         self.assertExportEqual(export, [
             ['Int Sum'      ,'Float Min'],
-            ['10 (2)'       ,'111.00'],
```

Comment:
Functionally this seems ok, but I don't understand why these tests had to be adapted (seems in the code like they should have proper decimals in place?)